admin/templates/configuration/tmp/configure/sql/qserv-worker.sql

```diff
@@ -1,6 +1,7 @@
--- Used by xrootd Qserv plugin:
+CREATE USER IF NOT EXISTS 'qsmaster'@'localhost';
```

Comment:
Why do you use "CREATE USER IF NOT EXISTS" here and "CREATE OR REPLACE USER" in qserv-czar.sql? Do you depend on the order in which these two scripts are executed?